src/styles/content-inspection.css

```
@@ -1012,6 +1020,12 @@ body:has(.fuma-content-inspection-detail) .hsas-admin-shell__content {
   letter-spacing: 0;
 }
 
+html[data-theme="dark"] .fuma-text-inspection-detail {
+  --text-inspection-border: var(--hsas-border);
+  --text-inspection-muted: var(--hsas-text-muted);
+  color: var(--hsas-text);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)